libsolidity/codegen/Compiler.h

```diff
@@ -78,8 +78,7 @@ class Compiler
 	eth::AssemblyItem functionEntryLabel(FunctionDefinition const& _function) const;
 
 private:
-	bool const m_optimize;
-	unsigned const m_optimizeRuns;
+	OptimiserSettings const m_optimiserSettings;
```

Comment:
Nice! So this is a write-once const, the constructor can set it ?

Reply:
Erm, can't remember having written this, but yes: const members can only be written to by the constructor.